realn.c

```diff
@@ -106,10 +106,29 @@ static int realn_check_tag(const uint8_t *tg, enum htsLogLevel severity,
 int sam_prob_realn(bam1_t *b, const char *ref, hts_pos_t ref_len, int flag)
 {
     int k, bw, y, yb, ye, xb, xe, apply_baq = flag&1, extend_baq = flag>>1&1, redo_baq = flag&4, fix_bq = 0;
+    enum htsRealnPlatform system = (flag>>3)<<3;
     hts_pos_t i, x;
     uint32_t *cigar = bam_get_cigar(b);
     bam1_core_t *c = &b->core;
-    probaln_par_t conf = { 0.001, 0.1, 10 };
+
+    // d(I) e(M) band
+    probaln_par_t conf = { 0.001, 0.1, 10 }; // Illumina
+
+    if (b->core.l_qseq > 1000 || system > BAQ_illumina) {
+        // Params that work well on PacBio CCS 15k.  Unknown if they
+        // help other long-read platforms yet, but likely better than
+        // the short-read tuned ones.
+        //
+        // This function has no access to thinks like the SAM header.
```

Comment:
```suggestion
        // This function has no access to things like the SAM header.
```

I guess in the fullness of time it might be worth having an HTSlib function that took an `@RG` header and returned a `sam_prob_realn()` flags value…